graylog2-bootstrap/pom.xml

```diff
@@ -80,6 +80,12 @@
     </dependencies>
 
     <build>
+        <resources>
```

Comment:
Is this necessary? A similar section (filtering properties files) is already in the parent POM.